src/reducers/events.js

```diff
@@ -18,7 +20,19 @@ export default function events(state = initialState, action = {}) {
       return Object.assign({}, state, {
         eventsLoading: false,
         eventsLoadingFailed: false,
-        allEvents: action.payload.events.sort(sortByDate),
+        allEvents: action.payload.events.sort(sortByDate).map(
+          (event) => {
+            const i = _.indexOf(state.favoriteEvents, event._id);
```

Comment:
Since you are doing this multiple times in the file and already imported lodash, you might just as well use _.includes directly in the if-statement: https://lodash.com/docs/4.16.1#includes